src/patch-accessors.js

```diff
@@ -170,6 +170,32 @@ let OutsideAccessors = {
 
 };
 
+const FormAssociatedAccessors = {
```

Comment:
We also need to fix <form> element so that it doesn't act on elements that are not in the same root.